task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -141,6 +144,38 @@ spec:
           name: dbfolder
         - mountPath: /work
           name: work
+    - name: upload
+      image: quay.io/konflux-ci/oras:latest@sha256:55cf304549225435958798c3be0377faf0a3ff89f3fbe03e45c3bb656ecaf02f
+      env:
+        - name: IMAGE_URL
+          value: $(params.image-url)
+        - name: IMAGE_DIGEST
+          value: $(params.image-digest)
+      workingDir: /work
+      script: |
+        #!/usr/bin/env bash
+
+        IFS=':' read -ra REPO <<< "$IMAGE_URL"
+        REPO="${REPO[0]}"
```

Comment:
This won't work if the IMAGE_URL has a port for the registry defined.

Reply:
Dropped this.